main.nf

```diff
@@ -175,41 +208,42 @@ process star {
     cpus 8
     memory '64 GB'
     time '5h'
-
+    
+    publishDir "$results_path/STAR"
+    
     input:
     file index
     file gtf
-    file trimmed_read1 from trimmed_read1
-    file trimmed_read2 from trimmed_read2
-
+    file (reads:'*') from trimmed_reads
+    set val(prefix) from name_for_star 
     output:
-    file '*.Aligned.sortedByCoord.out.bam' into bam4, bam5, bam6, bam7, bam8, bam9
-    file '*.Log.final.out' into results
-    file '*.Log.out' into results
-    file '*.Log.progress.out' into results
-    file '*.SJ.out.tab' into results
-
+    file '*.bam' into bam_rseqc, bam_preseq, bam_markduplicates, bam7, bam_featurecounts, bam_stringtieFPKM
```

Comment:
Seems like I missed to change that one. Now correctly named bam_md

On Wed, Apr 27, 2016 at 12:32 PM, Phil Ewels <notifications@github.com>
wrote:

> In main.nf
> <https://github.com/SciLifeLab/NGI-RNAseq/pull/5#discussion_r61235945>:
>
> >      output:
> > -    file '*.Aligned.sortedByCoord.out.bam' into bam4, bam5, bam6, bam7, bam8, bam9
> > -    file '*.Log.final.out' into results
> > -    file '*.Log.out' into results
> > -    file '*.Log.progress.out' into results
> > -    file '*.SJ.out.tab' into results
> > -
> > +    file '*.bam' into bam_rseqc, bam_preseq, bam_markduplicates, bam7, bam_featurecounts, bam_stringtieFPKM
>
> Is bam7 not used? Can we just remove it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/NGI-RNAseq/pull/5/files/6fbe6224f07b2b2dd9801a101a2dbc52d63b3c6d#r61235945>
>